VideoLocker/src/main/java/org/edx/mobile/util/MediaConsentUtils.java

```diff
@@ -14,6 +16,8 @@
  */
 public class MediaConsentUtils {
 
+    @Inject Config config;
```

Comment:
I would remove the Injected field and leave these methods static; use method parameters if you need the `Config`